academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -0,0 +1,385 @@
+
+    ```cmd
+    kesm.exe BuildIndexLocal --SchemaFilePath samplePrivateLibraryData.linked.schema.json --EntitiesFilePath samplePrivateLibraryData.linked.json --OutputIndexFilePath samplePrivateLibraryData.linked.kes --IndexDescription "Linked Private Library Publications"
+    ```
+
+>[!NOTE]
+> BuildIndexLocal command is only avaliable on win-x64 version of kesm
+
+
+Validate the index is built according to schema by inspecting the index meta data using the following command:
+
+```cmd
+kesm.exe DescribeIndex --IndexFilePath samplePrivateLibraryData.linked.kes
+```
+
+![describe index command output](media/snap-shot-describe-index-cmd-output.png)
+
+### Submit a index job for production workflow
+
+The index we're creating for this tutorial is relatively small and can be built locally. For larger and more complex index, use cloud builds to leverage high performing machines in Azure to build. To learn more, follow [How to create index from MAG](how-to-create-index-from-mag.md)
+
+## Deploy MAKES API Host with a custom index
+
+We are now ready to set up a MAKES API instance with a custom index.
+
+1. Upload the built, custom index to your MAKES storage account. You can do so by using following [Blob Upload from Azure Portal](https://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-portal). If you use cloud index build, you may skip this step.
+
+1. Run CreateHostResources to create a MAKES hosting virtual machine image.
+
+    ```cmd
+    kesm.exe CreateHostResources --MakesPackage https://<makes_storage_account_name>.blob.core.windows.net/makes/<makes_release_version> --HostResourceName <makes_host_resource_name>
+    ```
+
+> [!NOTE]
+> If your account is connected to multiple Azure Directories or Azure Subscriptions, you'll also have to specify the **--AzureActiveDirectoryDomainName** and/or **--AzureSubscriptionId** parameters. See [Command Line Tool Reference](reference-makes-command-line-tool.md#common-azure-authentication-parameters) for more details.
+
+1. Run DeployHost command and use the "--MakesIndex" parameter to load the custom linked private library publication index we've built.
+
+    ```cmd
+     kesm.exe DeployHost --HostName "<makes_host_instance_name>" --MakesPackage "https://<makes_storage_account_name>.blob.core.windows.net/makes/<makes_release_version>/"  --MakesHostImageId "<id_from_previous_command_output>" --MakesIndex "<custom_index_url>"
+    ```
+
+For more detailed deployment instructions, See [Create API Instances](get-started-create-api-instances.md#create-makes-hosting-resources)
+
+> [!NOTE]
+> Since the index we're hosting is relatively small, you can reduce Azure consumption for the tutorial MAKES host instance by using the "--HostMachineSku" parameter and set the SKU to "Standard_D2_V2".
+
+## Create Client Application with MAKES REST APIs
```

Comment:
>MAKES [](start = 34, length = 5)

Might want to take a beat and describe the application that we will be building as an example for the custom index.